server/athletics.py

```diff
@@ -0,0 +1,37 @@
+from flask import jsonify
+from base import *
+from server import app
+from pennathletics import athletes
+from flask import request
+from pennathletics import sportsdata
+from pennathletics import scraper
```

Comment:
`from pennathletics import athletes, sportsdata, scraper`
`from flask import request, jsonify`
Is your Python linter still working?